test/Data/Yaml/Marked/DecodeSpec.hs

```diff
@@ -32,7 +34,7 @@ data ExtraDep
   deriving stock (Eq, Show)
 
 decodeExtraDep :: Marked Value -> Either String (Marked ExtraDep)
-decodeExtraDep x = (fmap Git <$> decodeGitCommit x) <!> (fmap Plain <$> text x)
+decodeExtraDep x = (fmap Plain <$> text x) <!> (fmap Git <$> decodeGitCommit x)
```

Comment:
This just makes it so that, if the value is invalid for either, we'll see the error from the attempted Git parse.